pkg/sdn/plugin/subnets.go

```diff
@@ -36,8 +36,9 @@ func (master *OsdnMaster) SubnetStartMaster(clusterNetwork []*net.IPNet, hostSub
 		}
 	}
 	var subnetAllocatorList []*netutils.SubnetAllocator
-	for _, cidr := range clusterNetwork {
-		subnetAllocator, err := netutils.NewSubnetAllocator(cidr.String(), hostSubnetLength, subrange)
+	for _, cn := range clusterNetworks {
+		log.Infof("KEYWORD: ClusterCIDR: %s, HostSubnetLength: %d", cn.ClusterCIDR.String(), cn.HostSubnetLength)
```

Comment:
I don't think we want to log here. I guess, you added this log entry for your testing?